.meta/sinks/clickhouse.toml

```diff
@@ -94,19 +94,8 @@ common = true
 examples = ["mydatabase"]
 description = "The database that contains the stable that data will be inserted into."
 
-[sinks.clickhouse.options.encoding]
-type = "table"
-common = true
-required = false
-description = "Customize how events are encoded."
-
-[sinks.clickhouse.options.encoding.children.timestamp_format]
-type = "string"
-common = true
-required = false
-default = "rfc3339"
-description = "How to format event timestamps. Formats such as unix can be parsed as a Clickhouse DateTime, however, this loses precision as DateTimes are defined in seconds."
-
-[sinks.clickhouse.options.encoding.children.timestamp_format.enum]
-rfc3339 = "Format as an RFC3339 string"
-unix = "Format as a unix timestamp, can be parsed as a Clickhouse DateTime"
+<%= render("_partials/_encoding.toml",
+  namespace: "sinks.clickhouse.options",
+  encodings: ["default"],
+  default: "default"
```

Comment:
This sink should not have an encoding option since I assume Clickhouse accepts a specific structured data insert format.

Reply:
Right, these ones you see with `default: "default"` don't actually need to be set, they just need to be present. Would you prefer I hide these from the docs entirely?

Reply:
Yes, since there is no benefit in alternate encoding formats. The user should also receive a configuration error if they try to add `encoding.type` to this sink.